arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -267,5 +268,199 @@ public virtual async Task<GetCollectionFiguresResponse> GetCollectionFiguresAsyn
                 throw await GetApiErrorException(response).ConfigureAwait(false);
             };
         }
+
+        /// <summary>
+        /// Get the checksum for a specific collection.
+        /// GET /_api/collection/{collection-name}/checksum
+        /// </summary>
+        /// <param name="collectionName">Name of the collection.</param>
+        /// <param name="query">Query options.</param>
+        /// <returns></returns>
+        public virtual async Task<GetChecksumResponse> GetChecksumAsync(string collectionName, GetChecksumQuery query = null)
+        {
+            if (string.IsNullOrWhiteSpace(collectionName))
+            {
+                throw new System.ArgumentNullException("collectionName", "collectionName is required");
+            }
+            string uriString = _collectionApiPath + "/" + WebUtility.UrlEncode(collectionName) + "/checksum";
+            if (query != null)
+            {
+                uriString += "?" + query.ToQueryString();
+            }
+            using (var response = await _transport.GetAsync(uriString).ConfigureAwait(false))
+            {
+                if (response.IsSuccessStatusCode)
+                {
+                    var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
+                    return DeserializeJsonFromStream<GetChecksumResponse>(stream);
+                }
+                throw await GetApiErrorException(response).ConfigureAwait(false);
+            }
+        }
+
+        /// <summary>
+        /// Load Indexes into Memory.
+        /// Caches all index entries of this collection into the main memory. 
+        /// Therefore it iterates over all indexes of the collection and 
+        /// stores the indexed values, not the entire document data, 
+        /// in memory.
+        /// PUT /_api/collection/{collection-name}/loadIndexesIntoMemory
+        /// </summary>
+        /// <param name="collectionName">Name of the collection.</param>
+        /// <returns></returns>
+        public virtual async Task<LoadIndexesIntoMemoryResponse> LoadIndexesIntoMemoryAsync(string collectionName)
```

Comment:
I think that's how we should name this method ideally, but given that we are already far into applying the convention of having the HTTP method as a suffix, maybe apply the convention here? `LoadIndexesIntoMemoryAsync` -> `PutLoadIndexesIntoMemoryAsync`

There are a few other methods in this class that also needs reviewing based on that convention, e.g. `RecalculateCountAsync`, `CompactCollectionDataAsync`, `DocumentShardResponse`, etc.

We could review methods naming in a separate issue if you think it is needed. Some of the names are admittedly a bit clunky because of the convention🤔

Reply:
Good point. I've made the changes to follow the naming convention.